react-client/src/Components/App.js

```diff
@@ -237,26 +238,25 @@ function App() {
     });
 
     document.addEventListener("mouseover", async (e) => {  
-      e.target.setAttribute('data-dom-lens-target', true);   // data attribute to use as reference in domUtils.isTrueTarget to prevent unnecessary additional e.target          
-      document.querySelectorAll(".focused-dom").forEach((elTarget, idx) => {
-        elTarget.classList.remove("focused-dom");
-      });
-
-      if (!containsBookmarkModule(e)) { 
+      e.target.setAttribute('data-dom-lens-target', true);   // data attribute to use as reference in domUtils.isTrueTarget to prevent unnecessary additional e.target               
+      if (!containsBookmarkModule(e) && !containsSearchModule(e)) { 
+        
         if (!window.store.switchExtensionFunctionality || window.store.focusMode) return;
         if (focusMode) return;
 
         const isNotDomInfoComponent = !domUtils.ancestorExistsByClassName(e.target, "dom-info-dialog-box");
         const isNotBtnDisable = !domUtils.ancestorExistsByClassName(e.target, "dom-switch");
         const isNotSelectedDomFromBookmark = !domUtils.ancestorExistsByClassName(e.target, 'selected-dom');
+        const isNotSelectedFromSearch = !domUtils.ancestorExistsByClassName(e.target, 'search-panel');
+
         
-        if (isNotDomInfoComponent && isNotBtnDisable && e.target.nodeName !== "HTML" && isNotSelectedDomFromBookmark && !focusMode) {
+        if (isNotDomInfoComponent && isNotBtnDisable && e.target.nodeName !== "HTML" && isNotSelectedDomFromBookmark && isNotSelectedFromSearch && !focusMode) {
           const domType = e.target.nodeName?.toLowerCase();
-          await setDomLeanDetails({
+           setDomLeanDetails({
```

Comment:
why await is removed?

Reply:
already returned unintentionally removed